src/test_fetch_uri.c

```diff
@@ -97,9 +101,14 @@ static void test_fetch_http_nofragment_success(void) {
     g_object_unref(file);
 
     file = g_file_get_child(base, "runtest.sh");
-    g_assert(g_file_query_exists (file, NULL) != FALSE);
+    g_assert (g_file_query_exists (file, NULL) != FALSE);
+    file_info = g_file_query_info (file, "unix::mode", G_FILE_QUERY_INFO_NONE, NULL, NULL);
```

Comment:
If you’re only querying the one attribute (and not doing things like `unix::*`), using `G_FILE_ATTRIBUTE_UNIX_MODE` would be more consistent and clear.

Reply:
Yeah I felt same.